scl/elasticsearch/plugin.conf

```diff
@@ -20,7 +20,6 @@
 #
 #############################################################################
 
-@requires json-plugin
```

Comment:
Please do not remove this, either keep it here or push down into the block.

Reply:
again, this one is pending on the discussion on making json-plugin mandatory. resolving this.